tests/test_gil_scoped.py

```diff
@@ -181,7 +182,7 @@ def _run_in_process(target, *args, **kwargs):
         elif process.exitcode is None:
             assert t_delta > 0.9 * timeout
             msg = "DEADLOCK, most likely, exactly what this test is meant to detect."
-            if SKIP_IF_DEADLOCK:
+            if SKIP_IF_DEADLOCK or (env.PYPY and env.WIN):
```

Comment:
nit (readability) Why not hoist this logic up to the top? e.g.
```py
if env.PYPY or env.WIN:
   SKIP_IF_DEADLOCK = True  # See PR #4216
else:
   SKIP_IF_DEADLOCK = False
```

Reply:
I removed it completely: 29cea64554c19d5d7bc412d46c0b39d1c9705bd7

Reply:
Yay, thanks!